docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,317 @@
+```
+
+## OracleDaemonConfig
+
+```python
+NORMALIZED_CL_REWARD_PER_EPOCH=64
+NORMALIZED_CL_REWARD_MISTAKE_RATE_BP=1000
+REBASE_CHECK_NEAREST_EPOCH_DISTANCE=1
+REBASE_CHECK_DISTANT_EPOCH_DISTANCE=23  # 10% of AO 255 epochs frame
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200 # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800 # 4 days
+PREDICTION_DURATION_IN_SLOTS=50400
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100 # 1% network penetration for a single NO
+```
+
+## OracleReportSanityChecker
+
+```python
+churnValidatorsPerDayLimit = 40000
+oneOffCLBalanceDecreaseBPLimit = 500
+annualBalanceIncreaseBPLimit = 1000
+simulatedShareRateDeviationBPLimit = 50
+maxValidatorExitRequestsPerReport = 500
+maxAccountingExtraDataListItemsCount = 500
+maxNodeOperatorsPerExtraDataItemCount = 100
+requestTimestampMargin = 7680
+maxPositiveTokenRebase = 750000
+```
+
+## Burner
+
+```python
+# proofs:
+# - https://vote.lido.fi/vote/106
+# - https://etherscan.io/address/0xB280E33812c0B09353180e92e27b8AD399B07f26#readContract#F7
+totalNonCoverSharesBurnt = 32145684728326685744
+```
+
+## DepositSecurityModule
+
+```python
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F10
+maxDepositsPerBlock = 150
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F11
+minDepositBlockDistance = 25
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F14
+pauseIntentValidityPeriodBlocks = 6646
+```
+
+## AccountingOracle
+
+```python
+epochsPerFrame = 225  # once per day
+# So, the AccountingOracle expected report time would be 12:00 UTC
+
+fastLaneLengthSlots = 10
+```
+
+## ValidatorsExitBusOracle
+
+```python
+epochsPerFrame = 75  # thrice per day
+# So, the ValidatorsExitBusOracle expected report time would be 
+# 4:00 UTC, 12:00 UTC, 20:00 UTC
+
+fastLaneLengthSlots = 10
+```
+
+## GateSeal
+
+```python
+seal_duration = 518400  # 6 days as seconds (2 x gov time)
+expiry_timestamp = 1714521600  # 2024-05-01 00:00 UTC
+```
+
+## Roles setup
+
+### **Lido**
+*Aragon app*
+
+**New**
+- `UNSAFE_CHANGE_DEPOSITED_VALIDATORS_ROLE`
+    - None
+
+**Kept**
+- `PAUSE_ROLE`
+    - Voting
+- `RESUME_ROLE`
+    - Voting
+- `STAKING_PAUSE_ROLE`
+    - Voting
+- `STAKING_CONTROL_ROLE`
+    - Voting
+
+**Obsolete**
+- `MANAGE_FEE` `[MN, GL]`
+- `MANAGE_WITHDRAWAL_KEY` `[MN, GL]`
+- `MANAGE_PROTOCOL_CONTRACTS_ROLE` `[MN, GL]`
+- `BURN_ROLE` `[MN, GL]`
+- `DEPOSIT_ROLE` `[MN, GL]`
+- `SET_EL_REWARDS_VAULT_ROLE` `[MN, GL]`
+- `SET_EL_REWARDS_WITHDRAWAL_LIMIT_ROLE` `[MN,GL]`
+
+### **NodeOperatorsRegistry**
+*Aragon app*
+
+**New**
+- `STAKING_ROUTER_ROLE`
+    - StakingRouter (set in voting script)
+- `MANAGE_NODE_OPERATOR_ROLE`
+    - None (maybe Voting once upgrade applied?)
+
+**Kept**
+- `MANAGE_SIGNING_KEYS`
+- `SET_NODE_OPERATOR_LIMIT_ROLE`
```

Comment:
'Voting' and 'Easytrack Evmscriptexecutor' has role SET_NODE_OPERATOR_LIMIT_ROLE after votes. 
Might it be worth writing it down here?

Reply:
It's essential, indeed

Reply:
Added